src/__tests__/imagings/requests/NewImagingRequest.test.tsx

```diff
@@ -204,12 +213,18 @@ describe('New Imaging Request', () => {
         onChange({ currentTarget: { value: expectedImaging.type } })
       })
 
-      const statusSelect = wrapper.find(SelectWithLabelFormGroup)
+      const statusSelect = wrapper.find('.imaging-status').find(SelectWithLabelFormGroup)
```

Comment:
Just curious why you decided to use the class here? This test would have to be changed if for some reason the class name changes

Reply:
For my point of view, since there are two fields (Selecting status and visit) using SelectWithLabelFormGroup in the new request imaging page, I need to distinguish the two fields with different div classNames (in this case, imaging-status and imaging-visit) so that it is easier to find the component for testing